tutorials/mongoose/findupdatedoc.md

```diff
@@ -0,0 +1,16 @@
+To make `findOneAndUpdate()` return the updated document, you need to use the `returnDocument` option.
+`returnDocument` has two options, `'before'` and `'after'`, with the default behavior being it gives the document before the update.
+
+```javascript
+await Model.findOneAndUpdate(filter, update, {returnDocument: 'after'});
```

Comment:
More concrete example please. Show an example of creating a document, then updating it using `findOneAndUpdate()` and showing the difference between `returnDocument: 'before'` and `returnDocument: 'after'`